content/en/docs/concepts/services-networking/_index.md

```diff
@@ -49,5 +49,13 @@ blind to the existence or non-existence of host ports.
 Kubernetes networking addresses four concerns:
 - Containers within a Pod [use networking to communicate](/docs/concepts/services-networking/dns-pod-service/) via loopback.
 - Cluster networking provides communication between different Pods.
-- The [Service resource](/docs/concepts/services-networking/service/) lets you [expose an application running in Pods](/docs/concepts/services-networking/connect-applications-service/) to be reachable from outside your cluster.
-- You can also use Services to [publish services only for consumption inside your cluster](/docs/concepts/services-networking/service-traffic-policy/).
+- The [Service](/docs/concepts/services-networking/service/) resource lets you
+  [expose an application running in Pods](/docs/tutorials/services/connect-applications-service/)
+  to be reachable from outside your cluster.
+  - [Ingress](/docs/concepts/services-networking/ingress/) provides extra functionality
+    specifically for exposing HTTP applications, websites and APIs.
+- You can also use Services to
+  [publish services only for consumption inside your cluster](/docs/concepts/services-networking/service-traffic-policy/).
```

Comment:
Although this statement is true, it's not directly tied to External and Internal Traffic Policy configuration. Those policies affect traffic coming from inside or outside, but they don't need to be set to "publish services only for consumption inside your cluster". That specific statement sounds like it's describing ClusterIP Services.

Reply:
What's the right statement? It sounds like the original text was wrong.

Reply:
what about

> You can also influence the consumption of Services  [using traffic policies](/docs/concepts/services-networking/service-traffic-policy/).